packages/consumption/src/modules/notifications/NotificationsController.ts

```diff
@@ -34,7 +34,7 @@ export class NotificationsController extends ConsumptionBaseController {
     }
 
     public async sent(message: Message): Promise<LocalNotification> {
-        if (!message.isOwn) throw new Error("Cannot send a Notification from a foreign message.");
+        if (!message.isOwn) throw new Error("Cannot save a sent LocalNotification from a received Message.");
```

Comment:
This is a little bit confusing, maybe rephrase: "Cannot save a LocalNotification with status 'Sent' from a received Message."